extensions/wikia/Email/Controller/CategoryAddController.class.php

```diff
@@ -39,10 +39,6 @@ protected function assertValidCategoryPage() {
 		if ( !$this->categoryPage instanceof \Title ) {
 			throw new Check( "Invalid value passed for categoryPageId (param: childArticleId)" );
 		}
-
-		if ( !$this->categoryPage->exists() ) {
-			throw new Check( "Category Page doesn't exist." );
-		}
```

Comment:
Note that we don't need the category page to "exist".  For any MW article, ```exists``` means that someone has gone to the article page and made at least one edit.  You can add a page to any category you want without actually going to Category:SomeCat and editing it.  When you visit Category:SomeCat, it will tell you that the page needs content, but will show you all the pages that have been added to that category.